src/io/flutter/actions/OpenInAndroidStudioAction.java

```diff
@@ -54,21 +55,22 @@ public void actionPerformed(@NotNull AnActionEvent event) {
     }
     final String androidStudioPath = findAndroidStudio(event.getProject());
     if (androidStudioPath == null) {
-      FlutterMessages.showError("Unable to locate Android Studio",
-                                "You can configure the Android Studio location via " +
-                                "'flutter config --android-studio-dir path-to-android-studio'.");
+      FlutterMessages.showError(
+        "Unable to locate Android Studio",
+        "You can configure the Android Studio location via 'flutter config --android-studio-dir path-to-android-studio'.",
+        event.getProject());
```

Comment:
If enough time has elapsed, `event.getProject()` will return null. It's better to extract it at the beginning of the method, for use later on.

Reply:
Interesting; will do.